layout.tt

```diff
@@ -64,6 +59,9 @@ BLOCK navigationLink %]
           [% INCLUDE navigationLink href="community.html" title="Community" %]
           [% INCLUDE navigationLink href="governance.html" title="Governance" %]
           [% INCLUDE navigationLink href="donate.html" title="Donate" %]
+          [% WRAPPER navigationLink class="search" title="Donate" %]
```

Comment:
`class="search" title="Donate"` that seem wrong

Reply:
nice catch, fixed

Reply:
oops!